Makefile

```diff
@@ -78,6 +78,8 @@ test-skopeo:
 		git clone -b $(SKOPEO_BRANCH) https://github.com/$(SKOPEO_REPO) $${skopeo_path} && \
 		rm -rf $${vendor_path} && cp -r . $${vendor_path} && rm -rf $${vendor_path}/vendor && \
 		cd $${skopeo_path} && \
+		echo == Patching skopeo, badly - DO NOT MERGE Makefile changes == && \
+		sed -i -e "s:ctx, manifest:ctx, manifest, nil:" cmd/skopeo/layers.go && \
```

Comment:
`sed -i` won't build on OSX, it will take `-e` as the filename extension as the back copy to create.

Reply:
Not to worry.  This commit isn't something we want to keep, but in the meantime it lets us run the skopeo parts of the tests in CI.